examples/storage/couchdb/couchdb-petset.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: "apps/v1alpha1"
+kind: PetSet
+metadata:
+  name: kubetest001
+  namespace: default
+spec:
+  serviceName: kubetest001-backend
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
+            {
+            "name": "configer-couch",
+            "image": "glyndwr/kube-init-couchdb",
```

Comment:
Sure you can see how this image is build in `examples/storage/couchdb/images/init_couch/`